### PR TITLE
To Do-List structure

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
In this milestone:

- I use Javascript ES6 and Webpack;
- I build HTML and CSS of the project;
- I set up HTML&CSS, Javascript linter configs;
- I use GitHub flow;

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "bugs": {
+    "url": "https://github.com/elyor-doniyorov/ToDo-List-redo/issues"
+  },
+  "homepage": "https://github.com/elyor-doniyorov/ToDo-List-redo#readme",
+  "devDependencies": {
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
+    "webpack": "^5.46.0",
+    "webpack-cli": "^4.7.2"
```
Comment (bellom) [inline package.json]: Let's improve the webpack setup.
For using 'serve' command you need to install: 'webpack-dev-server' package.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const implementToDoItems = () => {
+  toDoItems.forEach((item) => {
+    listWrap.innerHTML += `<li class="list-item" data-key="${item.index}">
+      <div class="list-item-check-name">
+        <button type="button" class="check-item" data-key="${item.index}"></button>
+        <input class="description-input" type="text" value="${item.description}">
+      </div>
+      <button data-key="${item.index}" class="move-item"><i class="fas fa-ellipsis-v"></i></button>
+    </li>`;
```
Comment (bellom) [inline src/index.js:14]: In the UI, we need to add the following icons in the screenshot below to your app.
image

Kindly check the [project design](https://www.youtube.com/watch?v=AcUd-_Yjjqg) one more time.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @elyor-doniyorov  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi @elyor-doniyorov 👋 , 
## STATUS: APPROVED ✔️ 
You have done a great job implementing the changes requested by the previous reviewer. 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
#### OPTIONAL:
Please update the live demo link in your README file to keep your repository professional. Keep rocking 🚀 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**